paper/main.bib

```diff
@@ -53,7 +67,7 @@ @misc{Pyth03
 	year = {2003}}
 
 @misc{cond15,
-	author = {conda-forge community},
+	author = {{Conda-forge~community}},
```

Comment:
conda-forge intentionally stylizes the "author" name with a lowercase "C" https://conda-forge.org/docs/user/introduction.html#how-can-i-give-credit-to-conda-forge